### PR TITLE
As a user I should be able to confirm my account after signing up

### PR DESCRIPTION
**Pull request by keinydev**
As a user I should be able to confirm my account after signing up in order to ensure that all the user's email are valid

[Trello card here](https://trello.com/c/gyax34HS/16-3-as-a-user-i-should-be-able-to-confirm-my-account-after-signing-up-in-order-to-ensure-that-all-the-users-email-are-valid)

### CODE REVIEW

File: app/controllers/api/v1/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class ConfirmationsController < DeviseTokenAuth::ConfirmationsController
+      
+      def show
+        render json: {
+          status: 200,
+          message: "Your account has been confirmed"
```
Comment (sebastiancaraballo) [inline app/controllers/api/v1/confirmations_controller.rb:8]: Try using `i18n` here.

File: spec/requests/api/v1/sessions/create_spec.rb [hunk 2]
```diff
@@ -15,11 +15,28 @@
...
         signin
       end
 
+      it 'returns unathorized status 401' do
+        expect(response).to have_http_status(401)
+      end  
+
+      it 'returns error message' do
+        parsed_response = JSON.parse(response.body)
+        expect(parsed_response['errors'][0]).to match('A confirmation email was sent to your account at')
```
Comment (sebastiancaraballo) [inline spec/requests/api/v1/sessions/create_spec.rb:30]:
```suggestion
        expect(json[:errors].first).to match('A confirmation email was sent to your account at')
```

File: config/environments/test.rb [hunk 3]
```diff
@@ -42,6 +42,8 @@
   # ActionMailer::Base.deliveries array.
   config.action_mailer.delivery_method = :test
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (sebastiancaraballo) [inline config/environments/test.rb:45]: What about adding Sendgrid settings for production?

**Review by sebastiancaraballo — APPROVED** [review]

File: config/environments/development.rb [hunk 4]
```diff
@@ -35,6 +35,10 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { :address => "localhost", :port => 1025 }
```
Comment (santir489) [inline config/environments/development.rb:40]: prefer this syntax
` { address: "localhost", port: 1025 } `
 as line 38 😁

**Review by santir489 — APPROVED** [review]

**Pull request merged**